STM32H7B0VBTx_FLASH.ld

```diff
@@ -143,6 +144,9 @@ __CONFIGFLASH_START__ = __SAVEFLASH_END__;
 __CONFIGFLASH_END__   = __CONFIGFLASH_START__ + __CONFIGFLASH_LENGTH__;
 __FBFLASH_START__ = __CONFIGFLASH_END__;
 __FBFLASH_END__   = __FBFLASH_START__ + __FBFLASH_LENGTH__;
+/* TODO: It would be nice to start the extflash with the filesystem so it starts at a constant location */
```

Comment:
it's probably actually best to put the filesystem at the very end of the chip, that way statically adding more ROMs in the future doesn't mess with the filesystem.